### DISCUSSION TITLE
Using command line to export svg from xopp

### DISCUSSION THREAD

**Discussion by peey** [xournalpp/xournalpp#2325] [Q&A] [answered]
I see that 
```
xournalpp --help
``` 
gives this as output

```
  -i, --create-img         Image output filename (.png / .svg)
```

This lead me to believe that I can do conversions via command line. But I can't figure out how to use this cli flag. Is this possible?

**Comment by rolandlo** [comment, marked as answer]
For example `xournalpp -i output.svg input.xopp` works.
